anki_addon/ui/common.py

```diff
@@ -26,9 +26,16 @@ def __init__(self, parent=None):
     def sizeHint(self):
         text = self.currentText()
         metrics = QFontMetrics(self.view().font())
```

Comment:
For me, `metrics = self.fontMetrics()` here works better

Another thing is that it would be better to have a pop-up a bit wider. Currently, it cuts text on the right both in card type and language selector.

Something like this:
```
    def showPopup(self):
        metrics = self.fontMetrics()
        max_width = max(
            metrics.horizontalAdvance(self.itemText(i))
            for i in range(self.count())
        )
        width = max(max_width, self.width())
        self.view().setMinimumWidth(width)
        super().showPopup()
```